tests/unit/fem/test_pa_kernels.cpp

```diff
@@ -520,17 +520,17 @@ static void test_pa_integrator()
    GridFunction x(&fes), y_fa(&fes), y_pa(&fes);
    x.Randomize(1);
 
-   ConstantCoefficient pi(M_PI);
```

Comment:
If `M_PI` isn't used anymore, we can remove the `_WIN32` `cmath` `ifdef` statement at the top of the file.

Reply:
Done in 90669b2cd6cab5d687e9bb45dcdfe9b75e1d60c5